src/main/java/duke/command/EmptyDateTimeException.java

```diff
@@ -0,0 +1,13 @@
+package duke.command;
+
+import duke.DukeException;
+
+/**
+ * Represents an exception that occurs when the user does not provide a date and
+ * time for a deadline or event.
+ */
+public class EmptyDateTimeException extends DukeException {
```

Comment:
I like that the name of the exception clearly explains what the exception is for.